README.md

```diff
@@ -261,10 +261,10 @@ For login choose username `teamlead-1` with password `teamlead-1`.
 
 # ⚡️OpenEnergyBadge
```

Comment:
Section title should be adapted. [OpenEnergyBadge](https://www.green-coding.io/products/open-energy-badge/) is a project by Green Coding Solutions which is not directly related to the Blue Angel certification. Now that KADAI has the Blue Angel certification, it would make sense to prefer highlighting the Blue Engel certification over the OpenEnergyBadge. Or maybe choosing a more general section title.

Here some ideas for the section title:
- 🌱 Sustainability
- ⚡ Energy Efficiency
- 🌱 Blue Angel Certification & ⚡ OpenEnergyBadge